mobile-widgets/qml/DiveSummary.qml

```diff
@@ -181,7 +181,7 @@ Kirigami.ScrollablePage {
 			text: Backend.diveSummaryText[19]
 		}
 		TemplateLabel {
-			text: qsTr("avg. SAC")
+			text: qsTr("max. SAC")
```

Comment:
Does that mean we no longer present average sac ? That is bad, the average sac is used to see the a diver becomes better over timer (== consumes less air).

Reply:
please read the full patch :-(